test/nodes/test_extractor_translation.py

```diff
@@ -1,30 +0,0 @@
-import pytest
-
-from haystack.pipelines import TranslationWrapperPipeline, ExtractiveQAPipeline
-from haystack.nodes import DensePassageRetriever, EmbeddingRetriever
-from .test_summarizer import SPLIT_DOCS
-
-
-# Keeping few (retriever,document_store,reader) combination to reduce test time
-@pytest.mark.integration
-@pytest.mark.elasticsearch
-@pytest.mark.summarizer
-@pytest.mark.parametrize("retriever,document_store,reader", [("embedding", "memory", "farm")], indirect=True)
-def test_extractive_qa_pipeline_with_translator(
-    document_store, retriever, reader, en_to_de_translator, de_to_en_translator
-):
-    document_store.write_documents(SPLIT_DOCS)
-
-    if isinstance(retriever, EmbeddingRetriever) or isinstance(retriever, DensePassageRetriever):
-        document_store.update_embeddings(retriever=retriever)
-
-    query = "Wo steht der Eiffelturm?"
-    base_pipeline = ExtractiveQAPipeline(retriever=retriever, reader=reader)
-    pipeline = TranslationWrapperPipeline(
```

Comment:
This test with the TranslationWrapperPipeline wasn't moved somewhere else but it was completely removed. Where are we now testing that the TranslationWrapperPipeline is compatible with other pipelines (passing queries, documents or answers to the translator)?

Reply:
Uh you're right!! Thank you, I didn't mean to delete it. I wonder where it disappeared, I was confident I copied it over :sweat_smile:

Reply:
Ah-ah, I found the reason: this is a change from the Translator tests PR that somehow made it here :sweat_smile: Reverting!